app/models/user.rb

```diff
@@ -1,2 +1,10 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes
+
```

Comment:
- [ ] You did well using the right column name for the `has_many :posts,` and ` has_many :comments,`, but we need to keep it consistent and use a ` foreign_key: 'author_id'` for the `has_many :likes` association too.